routes/index.js

```diff
@@ -55,8 +61,15 @@ router.get('/chat/:chatType/:chatId', function(req, res, next) {
 
 router.get('/user/:userId', function(req, res, next) {
   const { userId } = req.params;
-  chatName = userId
+  /* Make button open user profile if on Android */
+  let buttonTitle = 'Download Status'
+  let buttonUrl = 'https://status.im/get/'
+  if (utils.isAndroid(req.headers['user-agent'])) {
+    buttonTitle = 'Open in Status'
+    buttonUrl = `status-im://user/${userId}`
```

Comment:
This returns the behavior that we had :+1:
The old behavior had a problem. If the app is not installed, nothing happens. There is a way to do that in the android browser:
```
window.location = "app-uri";
setTimeout(function() {
  window.location = "download-uri"
}, 250);
```
https://stackoverflow.com/questions/41123611/open-an-app-ios-android-via-javascript-with-a-fallback-redirect-to-app-play-s

Not sure if we want to add it now.

Reply:
Are you suggesting that the button should not be an `<a>` tag but rather the click be handled by JS that will change `window.location` and after a timeout try to change it to download link?

I don't know how well that would work but I can try making a separate PR for that.

Reply:
Yes, I mean we could do that. Sure it can be added later